cmake/Find/FindLibcint.cmake

```diff
@@ -27,14 +28,24 @@ else()
 
     # Set the dynamic or static libraries
     if(EXISTS ${LIBCINT_PREFIX}/lib/libcint.so)
-        set(Libcint_LIBRARIES "${Libcint_LIBRARIES};${LIBCINT_PREFIX}/lib/libcint.so")
+        set(Libcint_LIBRARIES "${LIBCINT_PREFIX}/lib/libcint.so")
     else()
         if(EXISTS ${LIBCINT_PREFIX}/lib/libcint.dylib)
-            set(Libcint_LIBRARIES "${Libcint_LIBRARIES};${LIBCINT_PREFIX}/lib/libcint.dylib")
+            set(Libcint_LIBRARIES "${LIBCINT_PREFIX}/lib/libcint.dylib")
         else()
-            set(Libcint_LIBRARIES "${Libcint_LIBRARIES};${LIBCINT_PREFIX}/lib/libcint.a")
+            set(Libcint_LIBRARIES "${LIBCINT_PREFIX}/lib/libcint.a")
```

Comment:
I think I missed an if-case during my last PR

```cmake
if(EXISTS ${LIBCINT_PREFIX}/lib/libcint.a)
```

Otherwise, `libcint.a` gets set even though it may not be present.